### DISCUSSION TITLE
Money usage

### DISCUSSION THREAD

**Discussion by MrJackey** [smeas/Beer-and-Plunder#179] [General]
Should it really cost money to do stuff or should the limiting factors be time and players instead?

**Positives**
This would make the goal more intuitive due to the money only increasing towards the goal. 
This would also make the game easier to balance since we have less variables to keep in mind.

**Comment by DreamingJohnny** [comment]
Yeah, I do think this is a good idea, basically for the resons that you've already listed.

Of course, if time is the only factor, it means that the minigames surrounding table repairs and switching beer barrels probably needs to be updated. Seeing as before they were based on deciding when to spend the money, they will lack a bit of weight now.

Still, that does not need to be an argument against it, just something to keep in mind.
